[DX-1728]-Fixed 5.3 RNs (#5666)
TykTechnologies/tyk-docs: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md

### Upgrade Instructions

> If you are upgrading to 5.3.5, please follow the detailed [upgrade instructions](#upgrading-tyk).

Reviewer: ```suggestion
When upgrading to 5.6.1, please follow the [detailed upgrade instructions](#upgrading-tyk).
```

There are no deprecations in this release.

### Upgrade Instructions

> If you are upgrading to 5.3.4, please follow the detailed [upgrade instructions](#upgrading-tyk).

Reviewer: ```suggestion
When upgrading to 5.3.4 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

There are no deprecations in this release.

### Upgrade Instructions

> If you are upgrading to 5.3.3, please follow the detailed [upgrade instructions](#upgrading-tyk).

Reviewer: ```suggestion
When upgrading to 5.3.3 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

### Upgrade Instructions

> If you are upgrading to 5.3.2, please follow the detailed [upgrade instructions](#upgrading-tyk).

Reviewer: ```suggestion
When upgrading to 5.3.2 please follow the [detailed upgrade instructions](#upgrading-tyk).
```
